app/javascript/lib/standardSetup.js

```diff
@@ -182,8 +182,21 @@ export default ({ players, provinceNames }) => {
     }
     provinces.set(province, { factory });
   }
+
+  const unitLimits = new Map();
+
+  for (const nation of Nation) {
+    if (nation == Nation.AH) {
+      unitLimits.set(nation, {armies: 10, fleets: 6});
+    } else if (nation == Nation.GB) {
```

Comment:
It shouldn't make much difference but for the sake of regularity, can we use `===` here? And on line 189?